### PR TITLE
fix(mme) removed a few compilation warnings

### PR DESCRIPTION
**Pull request by rdefosse**
Signed-off-by: Raphael Defosseux <raphael.defosseux@openairinterface.org>

## Summary

After the latest `master` branch merges, I saw a bunch of compilation warnings.

## Test Plan

Build and Ran on the OAI - MME build variant.
Tested w/ DsTester

## Additional Information

- [ ] This change is backwards-breaking

### CODE REVIEW

**Comment by ardzoht** [comment]
Can you please add to your description what you did to validate the change on the Test Plan section?

**Review by ardzoht — APPROVED** [review]
lgtm, please add to your test plan the validation steps for this change before landing, thanks!

File: lte/gateway/c/core/oai/tasks/mme_app/mme_app_sgs_reset.c [hunk 1]
```diff
@@ -93,7 +93,7 @@ status_code_e mme_app_handle_sgsap_reset_indication(
  **          Return:    RETURNok, RETURNerror                              **
  **                                                                        **
  ***************************************************************************/
-status_code_e mme_app_handle_reset_indication(
+bool mme_app_handle_reset_indication(
```
Comment (ardzoht) [inline lte/gateway/c/core/oai/tasks/mme_app/mme_app_sgs_reset.c:96]: Update documentation to reflect this is returning boolean now

**Review by ardzoht — CHANGES_REQUESTED** [review]

File: lte/gateway/c/core/oai/tasks/mme_app/mme_app_sgs_reset.c (hunk 1 above)
Comment (rdefosse) [reply lte/gateway/c/core/oai/tasks/mme_app/mme_app_sgs_reset.c:96]: Nice catch Alex!

**Review by ardzoht — APPROVED** [review]
lgtm, thanks for the change

**Pull request merged**